### PR TITLE
Created basic scoreboard sending and removing

### PR DESCRIPTION
**Pull request by Saiini**
Basic scoreboard sending and removing, this has no API at the moment

### CODE REVIEW

File: dragonfly/session/session.go [hunk 1]
```diff
@@ -161,6 +161,23 @@ func (s *Session) SendJukeBoxPopup(message string) {
...
 }
 
+// SendScoreboard ...
+func (s *Session) SendScoreboard(displayName string, objName string){
+	s.writePacket(&packet.SetDisplayObjective{
+		DisplaySlot:   "sidebar",
+		ObjectiveName: objName,
+		DisplayName:   displayName,
+		CriteriaName:  "dummy",
+		SortOrder:     0,
```
Comment (Sandertv) [inline dragonfly/session/session.go]: You can remove this line due to the default value. Also, have you checked if the "dummy" criteria name is required?

File: dragonfly/session/session.go [hunk 2]
```diff
@@ -161,6 +161,23 @@ func (s *Session) SendJukeBoxPopup(message string) {
...
+	})
+}
+// RemoveScoreboard ...
```
Comment (Sandertv) [inline dragonfly/session/session.go]: Please add a newline before this comment.

File: dragonfly/session/session.go (hunk 1 above)
Comment (Saiini) [reply dragonfly/session/session.go]: looking at other implementations - they used the dummy criteria name, hence why i added it there

Comment (Sandertv) [reply dragonfly/session/session.go]: Yeah I understand. Have you tried to see if it works when keeping this field empty as well?

Comment (Saiini) [reply dragonfly/session/session.go]: Just tried it, removing the sort order does nothing and allows the scoreboard to be displayed, but having the criteria name to nothing (or removing it entirely) doesn't display the scoreboard at all

File: dragonfly/session/session.go (hunk 2 above)
Comment (Saiini) [reply dragonfly/session/session.go]: fixed

File: dragonfly/session/session.go (hunk 1 above)
Comment (Sandertv) [reply dragonfly/session/session.go]: Okay, thanks for checking 👍

**Pull request merged**